common/config/rush/pnpmfile.js

```diff
@@ -40,6 +40,14 @@ function readPackage(packageJson, context) {
     packageJson.dependencies['typescript'] = '^2 || ^3';
   }
 
+  /**
+   * The below conditional is used to consolidate and ensure correct version of monaco-editor
```

Comment:
For this I was thinking something more like:
monaco-editor-webpack-plugin specifies a peer dependency on an earlier version of monaco-editor, but it seems to also work with the newer version we're using.